slipp/src/main/java/slipp/repository/UserDaoRepository.java

```diff
@@ -0,0 +1,30 @@
+package slipp.repository;
+
+import nextstep.stereotype.Repository;
+import slipp.dao.UserDao;
+import slipp.domain.User;
+
+import java.sql.SQLException;
+import java.util.List;
+
+@Repository
+public class UserDaoRepository implements UserRepository {
```

Comment:
https://stackoverflow.com/questions/8550124/what-is-the-difference-between-dao-and-repository-patterns/15513290